recipe/build.sh

```diff
@@ -10,13 +10,14 @@ if [ "${target_platform}" == "linux-ppc64le" ] || [ "${target_platform}" == "lin
 fi
 
 # Numpy cannot be found in ppc64le for some reason... some extra help will do ;)
+# "core" for np v1; "_core" for v2.
```

Comment:
Once you use `np.get_include()` you not longer need this since numpy manages it for you.

Reply:
```suggestion
```